rqalpha/mod/rqalpha_mod_sys_risk/validators/price_validator.py

```diff
@@ -32,8 +32,10 @@ def can_submit_order(self, order, account=None):
         limit_up = round(self._env.price_board.get_limit_up(order.order_book_id), 4)
         if order.price > limit_up:
             reason = _(
-                "Order Creation Failed: limit order price {limit_price} is higher than limit up {limit_up}."
+                "Order Creation Failed: {order_book_id} limit order price {limit_price} is higher "
```

Comment:
Order Creation Failed: limit order price {limit_price} is higher than limit up {limit_up}, order_book_id={order_book_id}